Expensify/App
# Added skin tone support for Emojis

@stitesExpensify Task complete. Can you review, please?

### Details
Extended the existing Emoji Picker to add Skin Tone support.

### Fixed Issues

$ #4323 

### Tests
1. Tested several emojis with all skin tones
2. Verified NVP is stored for Preferred Skin tone
3. Verified NVP is available on refresh on the same device and other devices
4. Tested Filter + Skintone
5. Verified missing skin tones for all the emojis
6. Tested default and null handling for emoji skin tones in case skin tone not available
7. UI elements in terms of selection, on hover, refreshing the list tested
8. UI matched with the mockup shared
9. Tested with Landscape mode for Mobile Devices



### QA Steps
1. Go to chats and open EmojiPicker
2. With the first attempt the default skin tone should show up.
3. Choose the skin tone and verify that all the Emojis with Skin tone support have the chosen skin tone
4. Select any emoji and it should render with the chosen skin tone
5. Change the skin tone and the emojis should have the updated skin tone
6. Login with another device and ensure that the chosen Emoji skin tone is reflected by default
7. Try deleting the emoji and see that it doesn't require two deletes. 
8. Attempt to load the Emoji picker in the Mobile landscape view as well and see that emoji picker doesn't break
9. Search the emojis with keywords. These should also show up the emojis in chosen skin tone only



### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/3069065/128575992-db9d5d94-10fd-42f8-8c48-c7903ea8657f.mp4

#### Mobile Web
https://user-images.githubusercontent.com/3069065/128576083-eddcdd13-7add-4c5a-9682-a309225fd048.mp4

<img width="748" alt="emoji-skintone-mweb-landscape" src="https://user-images.githubusercontent.com/3069065/128577164-e2131ece-76e1-4a67-9c58-0042998be35f.png">

**Updated**
<img width="722" alt="Screenshot 2021-09-09 at 9 19 51 AM" src="https://user-images.githubusercontent.com/3069065/132620632-d675ca8f-e25d-49a3-bd73-7afee8fce9e9.png">


#### Desktop
https://user-images.githubusercontent.com/3069065/128576109-651981a0-7d1b-463f-b168-1e1f263df712.mp4

#### iOS
https://user-images.githubusercontent.com/3069065/128576156-2787a270-bdec-4fad-9db2-c4240b5c5211.mp4
<img width="764" alt="emoji-skintone-ios-landscape" src="https://user-images.githubusercontent.com/3069065/128577479-26270f45-afd9-40e6-8b2f-cede0012d25b.png">

**Updated**
<img width="757" alt="Screenshot 2021-09-09 at 9 26 41 AM" src="https://user-images.githubusercontent.com/3069065/132620620-bc132b79-ea8b-4982-abf1-b531d427b1ae.png">


#### Android
https://user-images.githubusercontent.com/3069065/128576868-6800164f-5b8a-49a5-8495-793eff6a839c.mp4

![emoji-skintone-android-landscape](https://user-images.githubusercontent.com/3069065/128577119-8221f11b-3465-4ddd-9ae1-08cc034cddde.jpeg)

**Updated**
<img width="1105" alt="Screenshot 2021-09-09 at 9 32 11 AM" src="https://user-images.githubusercontent.com/3069065/132620607-7888cf4d-d3de-4bce-bd18-b52ba7040b80.png">
